src/RangeValidator.php

```diff
@@ -110,8 +110,8 @@ public function doValidation( $value ) {
 	 * @return boolean
 	 */
 	protected function validateBounds( $value, $upperBound = null, $lowerBound = null ) {
-		$upperBound = is_null( $upperBound ) ? $this->upperBound : $upperBound;
-		$lowerBound = is_null( $lowerBound ) ? $this->lowerBound : $lowerBound;
+		$upperBound = $upperBound == null ? $this->upperBound : $upperBound;
```

Comment:
This should be a strict check (three equal signs) to be a capable replacement for `is_null()`.

```suggestion
		$upperBound = $upperBound === null ? $this->upperBound : $upperBound;
```

Given we also see a PHP update in this chain of commits, a [null coalescing operator](https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op) may make things easier of the eyes.

```suggestion
		$upperBound = $upperBound ?? $this->upperBound;
```

Same goes for the next line.